Gems/PrefabDependencyViewer/Code/Source/DirectedGraph.h

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/containers/unordered_set.h>
+#include <AzToolsFramework/Prefab/Template/Template.h>
+#include <AzToolsFramework/Prefab/PrefabSystemComponentInterface.h>
+#include <AzToolsFramework/Prefab/PrefabDomTypes.h>
+#include <AzCore/std/algorithm.h>
+#include <AzCore/std/containers/vector.h>
+
+namespace PrefabDependencyViewer
+{
+    namespace Utils
+    {
```

Comment:
These nested namespaces can be collapsed.
```suggestion
namespace PrefabDependencyViewer::Utils
{
```

Reply:
Done